geomesa-tools/pom.xml

```diff
@@ -66,6 +66,30 @@
             <artifactId>commons-csv</artifactId>
         </dependency>
 
+        <!-- Kafka CLI dependencies -->
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-kafka-datastore</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.11</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j:*</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.xerial.snappy</groupId>
+                    <artifactId>snappy-java</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
Is this needed for geomesa tools or kafka? Note that hadoop needs version 11.0.2 on the classpath to work properly and mixing in a 2nd version can break things depending on what this is.